addons/l10n_lt/data/account_chart_template_setup_data.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <!-- Setup Chart of Accounts Template -->
+    <record id="account_chart_template_lithuania" model="account.chart.template">
+        <field name="property_account_receivable_id" ref="account_account_template_2410"/>
+        <field name="property_account_payable_id" ref="account_account_template_4430"/>
+        <field name="property_account_expense_categ_id" ref="account_account_template_6000"/>
+        <field name="property_account_income_categ_id" ref="account_account_template_5000"/>
+        <field name="income_currency_exchange_account_id" ref="account_account_template_5803"/>
+        <field name="expense_currency_exchange_account_id" ref="account_account_template_6803"/>
+        <field name="property_stock_account_input_categ_id" ref="account_account_template_2045"/>
+        <field name="property_stock_account_output_categ_id" ref="account_account_template_2045"/>
```

Comment:
`property_stock_account_...` fields are later used for filling account fields for new product category records. The account set here is `2045` - `Purchased Goods for Resale in Transit`, which is correct for companies that resell items. For Manufacturers, this might end up not being correct and will have to be changed every time on new product category creation.
Is this intended?

Reply:
I agree, its too specific. I guess I can change to `Goods in Transit` (`Prekės kelyje`). If someone needs more specific accounts, they can create accounts with `2045xxx` or something like that.

Reply:
On second thought, I rushed with an answer... I checked official COA sample and indeed same name is used. So I can't just rename it for more generic use. 
Also I don't see better alternatives, because they all serve specific purpose. If I would change for example to `2035`, it would maybe serve for manufacturing companies, but not for others.

This one looks like the most generic case. Keep in mind, it is only used to set default property in product category. Then you change per category if you need different account to be used (it might be one account on one category, but different on another. E.g one used for manufactured goods, another for raw materials).

And actually if I would use some generic account, it would probably not fit any case at all. So I think, this is the case, where you can't satisfy everyone and you just try to match most (or some) cases.

Reply:
I agree. It is specific, yet fits most companies. It is also possible to not even add these fields. That way, new categories will not have them autofilled. I just do not know what is prefered by Odoo. 
* To have something set that doesn't fit all cases
* To not have anything set, but then force users to set it themself.